### PR TITLE
Moves RFCs that have been implemented into the implemented list.

### PR DESCRIPTION
**Pull request by ForestEckhardt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,14 +16,18 @@
 * [0002: Paketo Governance](./text/0002-governance.md)
 * [0004: Leiningen (Clojure) Buildpack](./text/0004-clojure.md)
 * [0005: Ruby Paketo Buildpack Promotion](./text/0005-ruby-promotion.md)
+* [0006: Web Servers Buildpack Subteam](./text/0006-web-servers.md)
```
Comment (ryanmoran) [inline README.md:19]: I would argue this isn't done until https://github.com/paketo-community/web-servers/ has a valid `buildpack.toml` that produces a buildpack as outlined in the RFC.

File: README.md [hunk 2]
```diff
@@ -3,13 +3,9 @@
...
 
 * [0003: Replace buildpack.yml with Build Plan (TOML)](./text/0003-replace-buildpack-yml.md)
-* [0006: Web Servers Buildpack Subteam](./text/0006-web-servers.md)
 * [0010: Dependency Mappings](./text/0010-dependency-mappings.md)
-* [0014: Paketo Community Rust Buildpack](./text/0014-rust.md)
-* [0015: Distribute Buildpacks via Docker Hub](./text/0015-dockerhub-distribution.md)
 * [0017: Paketo Community Go HTTP Function Buildpack](./text/0017-go-http-fn.md)
 * [0019: Default Behaviour for Buildpack-Set Language Ecosystem Environment Variables](./text/0019-buildpack-set-env-vars-defaults.md)
-* [0021: Paketo Community Go Generate Buildpack](./text/0021-go-generate-buildpack.md)
 * [0022: Core-deps governance restructure proposal](./text/0022-core-deps-governance-restructure.md)
```
Comment (ryanmoran) [inline README.md:9]: This is done.

File: README.md [hunk 3]
```diff
@@ -3,13 +3,9 @@
...
 * [0023: Git Support](./text/0023-git-buildpack.md)
 * [0024: Utility Buildpacks Team](./text/0024-utility-buildpacks-team.md)
```
Comment (ryanmoran) [inline README.md:11]: This is done.

**Review by ryanmoran — CHANGES_REQUESTED** [review]
Can we also add the Emeritus Status RFC into the implemented section?

File: README.md [hunk 4]
```diff
@@ -20,14 +16,18 @@
...
 * [0004: Leiningen (Clojure) Buildpack](./text/0004-clojure.md)
 * [0005: Ruby Paketo Buildpack Promotion](./text/0005-ruby-promotion.md)
+* [0006: Web Servers Buildpack Subteam](./text/0006-web-servers.md)
 * [0007: Paketo Buildpack Naming](./text/0007-buildpack-naming.md)
 * [0008: Paketo Community](./text/0008-paketo-community.md)
 * [0009: Dep Server to provide buildpack dependency metadata](./text/0009-dep-server.md)
 * [0012: Builders Subteam](./text/0012-builder-subteam.md)
 * [0013: Contributor Guidelines](./text/0013-contributing-guidelines.md)
+* [0014: Paketo Community Rust Buildpack](./text/0014-rust.md)
+* [0015: Distribute Buildpacks via Docker Hub](./text/0015-dockerhub-distribution.md)
```
[the remark this answers is not in the document]
Comment (ForestEckhardt) [reply README.md:26]: Ok I will move that back.

**Review by ryanmoran — APPROVED** [review]

**Pull request merged**